draft-ietf-webpush-protocol.xml

```diff
@@ -1137,6 +1147,12 @@ HEADERS      [stream 82] +END_STREAM
           This could force the user agent to serialize subscription requests to the push
           service.
         </t>
+        <t>
+         A user agent that monitors subscriptions for other clients might

```

Comment:
This has been included in the draft for some time now with the 3rd paragraph of 4.1.
Anyway, the goal here is simply to ensure that a push service will not prevent a user agent from acting as a proxy by restricting too strictly the number of subscription sets that can be used by a user agent.